web app/src/main/java/de/tinf15b4/kino/web/ui/SmartCinemaUi.java

```diff
@@ -125,7 +128,8 @@ private Component createHeader() {
         search.addComponent(searchField);
         search.addComponent(searchButton);
         searchButton.addClickListener(
-                e -> getNavigator().navigateTo(SearchResultsView.VIEW_NAME + "/" + searchField.getValue()));
+                e -> getNavigator().navigateTo(
+                        SearchResultsView.VIEW_NAME + "/" + searchField.getValue()));
```

Comment:
Seems like you are using some wring formatting. There was no change in this line as far as I can see, but still there is a new formatting